charybdis-parser/src/fields.rs

```diff
@@ -107,6 +112,11 @@ impl<'a> Field<'a> {
         FieldAttributes::from_attributes(&field.attrs)
             .map(|char_attrs| {
                 let ignore = char_attrs.ignore.unwrap_or(false);
+
+                let column_type = char_attrs.column_type.clone().map(
+                    |tname| CqlType::from_str(tname.as_str()).unwrap()
+                ).unwrap_or_else(|| Field::outer_type(&field.ty, ignore));
```

Comment:
Here, we can do:

```rust
        let column_type =
                    char_attrs
                        .column_type
                        .as_ref()
                        .map_or(Field::outer_type(&field.ty, ignore), |tname| {
                            CqlType::from_str(tname.as_str()).expect(format!("Unknown column type: {}", tname).as_str())
                        });
```

So we can improve error handling. Otherwise, we will not have clue why migrations failed. Also we avoid clone.